src/components/ChatListWebchats.tsx

```diff
@@ -10,13 +10,7 @@ const ListWebchats: FC<{
   <>
     {p.datas.map(
```

Comment:
Filter first: `p.datas.filter(c => c.webchat).map(item => ...)`

Reply:
I did try but still got synatx of typescript 

![image](https://user-images.githubusercontent.com/5568604/119235875-bdc42380-bb5e-11eb-9dd7-2ee431393935.png)

So, we can keep current or make a new interface with webchat not _undefined_ 
if create new interface it look like that!
![image](https://user-images.githubusercontent.com/5568604/119235968-2dd2a980-bb5f-11eb-926a-8c977a7b6da3.png)

what's your idea ?